src/components/blocks/Endpoint/Endpoint.js

```diff
@@ -10,7 +10,7 @@ function Endpoint(props) {
 
   return (
     <>
-      <EndpointBlock>
+      <EndpointBlock className="rpg-endpoint-block">
```

Comment:
can change it to `rpg-block-endpoint`. In next PR, add a ability to configure the classNames